socorro/schemas/__init__.py

```diff
@@ -4,20 +4,22 @@
 
 import json
 
+import yaml
 from pkg_resources import resource_stream
 
 
-def _get_file_content(filename, parsed=True):
-    with resource_stream(__name__, filename) as f:
-        if parsed:
-            return json.load(f)
-        else:
-            return f.read()
+def get_file_content(filename):
+    if filename.endswith(".json"):
+        with resource_stream(__name__, filename) as fp:
+            return json.load(fp)
 
+    if filename.endswith(".yaml"):
+        with resource_stream(__name__, filename) as fp:
+            return yaml.load(fp, Loader=yaml.Loader)
 
-TELEMETRY_SOCORRO_CRASH_SCHEMA = _get_file_content("telemetry_socorro_crash.json")
-TELEMETRY_SOCORRO_CRASH_SCHEMA_AS_STRING = _get_file_content(
-    "telemetry_socorro_crash.json", parsed=False
-)
 
-JAVA_EXCEPTION_SCHEMA = _get_file_content("java_exception.json")
+PROCESSED_CRASH_SCHEMA = get_file_content("processed_crash.1.schema.yaml")
+
+TELEMETRY_SOCORRO_CRASH_SCHEMA = get_file_content("telemetry_socorro_crash.json")
+
+JAVA_EXCEPTION_SCHEMA = get_file_content("java_exception.json")
```

Comment:
This is now a subschema in the processed crash schema. At some point, I'll fix this so we only have the java exception stuff defined once.